### PR TITLE
Remove XMS prefix from header response fields

### PR DESCRIPTION
**Pull request by jhendrixMSFT**
Added Crc to list of TLAs for capitalization.

### CODE REVIEW

File: src/namer/namer.ts [hunk 1]
```diff
@@ -79,7 +79,6 @@ async function process(session: Session<CodeModel>) {
   // pascal-case and capitzalize acronym operation groups and their operations
   for (const group of values(model.operationGroups)) {
     const details = <Language>group.language.go;
-    const opGroupName = capitalizeAcronyms(pascalCase(group.$key));
```
Comment (catalinaperalta) [inline src/namer/namer.ts]: Why dont we want to have this naming fix anymore?

Comment (jhendrixMSFT) [reply src/namer/namer.ts]: It's dead code, but you bring up a good point that maybe it shouldn't be.  Let me take a look.

File: src/generator/operations.ts [hunk 2]
```diff
@@ -204,10 +204,10 @@ function formatParamValue(param: Parameter, imports: ImportManager): string {
 }
 
 // use this to generate the code that will help process values returned in response headers
-function formatHeaderResponseValue(header: string, schema: Schema, imports: ImportManager, respObj: string): string {
+function formatHeaderResponseValue(propName: string, header: string, schema: Schema, imports: ImportManager, respObj: string): string {
```
Comment (catalinaperalta) [inline src/generator/operations.ts]: What's the difference between propName and header?

Comment (jhendrixMSFT) [reply src/generator/operations.ts]: `propName` is the name of the property as it appears in the struct.  `header` is the actual header, e.g. `x-ms-foo`

Comment (catalinaperalta) [reply src/generator/operations.ts]: Ah I see. Thanks for the explanation!

File: src/namer/namer.ts (hunk 1 above)
Comment (jhendrixMSFT) [reply src/namer/namer.ts]: The reference to it was removed in https://github.com/Azure/autorest.go/pull/313.  Apparently I forgot to remove the definition then.  :(

**Pull request merged**